### PR TITLE
Adapt hocr-wordfreq for non-ASCII symbols

### PR DESCRIPTION
**Pull request by zuphilip**
This is tested with the ersch (fraktur) example from ocropy.

### CODE REVIEW

**Comment by stweil** [comment]
Interesting solution, looks good. Do we want to restrict words to letters and numbers (excluding other printable characters)?

File: hocr-wordfreq [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```
Comment (stweil) [inline hocr-wordfreq]: This is also used in all other scripts, good.

File: test/hocr-wordfreq/hocr-wordfreq.tsht [hunk 2]
```diff
@@ -9,5 +9,5 @@ casesensitive=$(hocr-wordfreq -i $TESTDATA/sample.html | head -n 1)
 exec_ok "hocr-wordfreq" "$TESTDATA/sample.html"
 exec_ok "hocr-wordfreq" "-i" "-n 30" "$TESTDATA/sample.html"
 
-match "24\s*the" "$normal" "check that no errors are found when comparing the sample file with itself"
-match "23\s*the" "$casesensitive" "check that no errors are found when comparing the sample file with itself"
+match "24\s*the" "$normal" "check the most frequent word in sample.html"
```
Comment (stweil) [inline test/hocr-wordfreq/hocr-wordfreq.tsht]: The `-i` issue should be handled in a separate commit. The comments here don't match the help text.

Comment (zuphilip) [inline test/hocr-wordfreq/hocr-wordfreq.tsht]: Okay, I will delete these changes here and we can continue discussion of these in #100

**Comment by zuphilip** [comment]
> Do we want to restrict words to letters and numbers (excluding other printable characters)?

Well, I want to exclude interpunctations and anything which does not look like a "word". Yes, to split on `\W+` the non-word characters does make sense for me. Do you encounter any problems with the current solution?
